### DISCUSSION TITLE
Questions about the bulk viscosity and TET10

### DISCUSSION THREAD

**Discussion by QuatorzeM** [OpenRadioss/OpenRadioss#1542] [Q&A] [answered]
Dear Community,

 As I posted last week, I am woking on the TET10 element of  OpenRadioss. I find something interesting to discuss.

1. Bulk viscosity is often used to increase the simulation stability. For elements possessing one intergration points, the length, the density of the whole element is used to calculate the bulk viscosity term, which is easy for implementation. 
But for elements with multi-intergration points, it seems not so clear. For example, TET10 have four intergration points, so the stress tensor will be calculated for 4 times. Every time the stress tensor is calculated, we need to calculate the bulk viscosity term. During this process, I am wondering how to define the characteristic length, the length of the whole element? or the length of the volume of this intergration points?  The same for density and wave speed, how we define it? 

2. the second question is about the plasticity for TET10, I noticed when Itetra10 = 1000，the icp will be 0, so no special method will be used for the incompressible problem. When Itetra10 = 2, the icp will be 2, so the incompressible problem is treated. I am wondering for this  two type of TET10, which method has a higher numerical stability, especially for low quality element? 

Best regards,
Fabien

**Comment by qiangRadioss** [comment, marked as answer]
Hi Fabien,

For 1). yes, the integration points are used to compute the internal force by integration numerically, the key is the sub-volume Vj in integral for your question of bulk viscosity term: the characteristic length (lc=(Vj)^(1/3)), density...
For 2) Icpre in /PROP/SOLID isn't used for T10, Itetra10 could be used, but it has no relation with Icpe.

hope this helps.
Qiang.

**Reply by QuatorzeM** [reply]
Thanks’s QIANG, I find the bulk viscosity is important for the numerical stability，expecially for tet elements. That helps a lot. 
(Forgert to respond you, because of covid.... )

Fabien

**Reply by elequiniou** [reply]
Hi Fabien, hope you are better now... I am really pleased to see your interest in element formulation in OpenRadioss.
If you have the opportunity, you are welcome to participate to the OpenRadioss Users Day and meet the community!!!
Registration info on the web site : openradioss.org
It is in-person free event, everyone is welcome to come, just need to register ;-)

**Reply by QuatorzeM** [reply]
Hey elequiniou,

Sorry to see you infomation so late, I am getting better.  The event interested me a lot but I see it's in Germany. I am in China now.
I'd like to know if the some events can be hold in China :). 
I'like to know the R&D center of Radioss is in France or Germany ? 

Best wishes,
Fabien

**Reply by elequiniou** [reply]
Hi Fabien,

We are looking at possibility to stream the event in addition to in-person for those who are a bit far away... So stay tuned ;-)
Altair have many offices all around the globe, including R&D centers. For Radioss historically the development is based in France, but we do have an international team, including Germany and other places worldwide. 
And now we have contributors from many places !!!

Kind Regards,
Eric

**Reply by QuatorzeM** [reply]
Hi elequiniou,

That's registered ;)

Best wishes,
Fabien
